### ISSUE TITLE
empty statement: unable to suppress

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#602]
```
procedure foo private:
   define input parameter i_c as character no-undo.

   case i_c:
      when "" then /* do not complain */ .
      when "bar":u then message 'bar':u.
      otherwise message 'error':u.
   end case.

end procedure.

```
I am unable to suppress the empty statement rule on the 'when "" then .'

**Comment by gquerret** [comment]
Rule now doesn't complain anymore if there's a comment in an empty `WHEN ... THEN .`
